rpcclient/mining.go

```diff
@@ -100,3 +105,55 @@ func (r FutureGetBlockTemplateResult) Receive() (*rpcmodel.GetBlockTemplateResul
 func (c *Client) GetBlockTemplate(payAddress string, longPollID string) (*rpcmodel.GetBlockTemplateResult, error) {
 	return c.GetBlockTemplateAsync(payAddress, longPollID).Receive()
 }
+
+// ParseBlock Accepts a GetBlockTemplateResult and parse it into a Block
+func ParseBlock(template *rpcmodel.GetBlockTemplateResult) (*util.Block, error) {
+	// parse parent hashes
+	parentHashes := make([]*daghash.Hash, len(template.ParentHashes))
+	for i, parentHash := range template.ParentHashes {
+		hash, err := daghash.NewHashFromStr(parentHash)
+		if err != nil {
+			return nil, errors.Errorf("Error decoding hash %s: %s", parentHash, err)
```

Comment:
Use errors.Wrap instead, and the go convention is to start errors with small case (Please fix it everywhere in this function as well)

Reply:
Done